src/utils/getFamilyMetadata.tsx

```diff
@@ -25,22 +26,27 @@ function displayConceptHierarchy(concept: TFamilyConceptTreeNode): React.ReactNo
 }
 
 // Format the family metadata into a shape suitable for the MetadataBlock component
-export const getFamilyMetadata = (family: TFamilyNew, countries: TGeography[]): IMetadata[] => {
+export const getFamilyMetadata = (family: TFamilyNew, countries: TGeography[], subdivisions: TGeographySubdivision[]): IMetadata[] => {
   const familyMetadata = [];
 
   // TODO: handle more categories and their specific metadata later
   if (family.corpus_type_name.toLowerCase() === "litigation") {
-    familyMetadata.push(...getLitigationMetaData(family, countries));
+    familyMetadata.push(...getLitigationMetaData(family, countries, subdivisions));
   }
 
   return familyMetadata;
 };
 
-function getLitigationMetaData(family: TFamilyNew, countries: TGeography[]): IMetadata[] {
+function getLitigationMetaData(family: TFamilyNew, countries: TGeography[], subdivisions: TGeographySubdivision[]): IMetadata[] {
   const metadata = [];
 
   // Structure concepts into a hierarchy we can use
   const hierarchy = buildConceptHierarchy(family.concepts);
+  const geosOrdered = family.geographies.sort((a, b) => {
```

Comment:
Suggestion: For a complex sort (especially if it would be beneficial to sort alphabetically _after_ length, consider [Lodash orderBy](https://lodash.com/docs/4.17.15#orderBy).

Reply:
Good shout on both lodash and sortby alophabetical